inc/config.class.php

```diff
@@ -2564,7 +2564,23 @@ static function checkWriteAccessToDirs($fordebug = false) {
                __('Checking write permissions for log files')."</td>";
       }
 
-      if (error_log("Test\n", 3, GLPI_LOG_DIR."/php-errors.log")) {
+      $oldhand = set_error_handler(function($errno, $errmsg, $filename, $linenum, $vars){return true;});
+      $oldlevel = error_reporting(0);
+
+      $can_write_logs = false;
+      if (Toolbox::testWriteAccessToDirectory(GLPI_LOG_DIR) === 0) {
```

Comment:
Nah, can't use `testWriteAccessToDirectory` for `GLPI_LOG_DIR`, will be blocked by SElinux

Reply:
oh! Good catch, thank you. Will try with bad selinux context as well.